feat: converted main.workflow to Actions V2 yml files (#19)
Decathlon/pull-request-labeler-action: README.md

<p align="center">
  <img src="https://github.com/Decathlon/pull-request-labeler-action/raw/master/assets/intro.png" alt="Result illustration"/><br>
  <a href="https://app.fossa.io/projects/git%2Bgithub.com%2FDecathlon%2Fpull-request-labeler-action?ref=badge_shield"><img src="https://app.fossa.io/api/projects/git%2Bgithub.com%2FDecathlon%2Fpull-request-labeler-action.svg?type=shield"></a>
  <a href="https://circleci.com/gh/Decathlon/pull-request-labeler-action/tree/master"><img src="https://circleci.com/gh/Decathlon/pull-request-labeler-action.svg?style=svg"></a>
  <br><br>

>   This repository provide a GitHub actions to <strong>auto label a pull request</strong> based on committed files.

Reviewer: ```suggestion
  This repository provides a GitHub action to <strong>automatically label a pull request</strong> based on committed files.
```

</p>

**Table of Contents**

- [Common usage](#common-usage)
- [Breaking change](#breaking-change)
- [Startup](#startup)
  - [Configuration](#configuration)
  - [Use Github action](#use-github-action)
    - [Settings for v1.0.0+ release (deprecated)](#settings-for-v100-release-deprecated)
    - [Settings for v2.0.0+ release](#settings-for-v200-release)
- [Contributing](#contributing)
  - [Commands](#commands)
- [License](#license)

> 

Reviewer: ```suggestion
```

> 

Reviewer: remove duplicated empty line

  <img src="https://github.com/Decathlon/pull-request-labeler-action/raw/master/assets/log.png" alt="Action log messages"/>
</p>

## Breaking change

> 

Reviewer: ```suggestion
```

> 

Reviewer: remove duplicated empty line
